tests/plugins/test_tag_and_push.py

```diff
@@ -81,6 +81,26 @@
 ]
 
 
+def get_repositories_annotations(tag_conf):
+    primary_repositories = []
+    for image in tag_conf.primary_images:
+        primary_repositories.append(image.to_str())
+
+    unique_repositories = []
+    for image in tag_conf.unique_images:
+        unique_repositories.append(image.to_str())
+
+    floating_repositories = []
+    for image in tag_conf.floating_images:
+        floating_repositories.append(image.to_str())
+
+    return {
+        "primary": primary_repositories,
+        "unique": unique_repositories,
+        "floating": floating_repositories,
+    }
```

Comment:
Would it be possible to hardcode expected value of the annotation? I'm not a big fan of unit tests that duplicate the business logic

Reply:
well tests in this plugin are skipped apart from one, so I can't really test it. So hardcoding can be done later

Reply:
Oh, does that test not produce the annotation? But we still use this plugin for source containers, and we need that annotation for source containers

Reply:
one test which is enabled produces annotation, others are disabled